### PR TITLE
Bump infoblox provider version to 1.1.1, terraform to 1.0+

### PR DESCRIPTION
**Pull request by broestls**
Updates the module with Terraform 1.0 series support as well as the provider version. 

Note that the provider has silently dropped the `network_view_name` provider between `1.0.4` and `1.1.1`. It is not documented why, but we also don't require a custom value here. Impact appears to be low for now.

### CODE REVIEW

**Review by mirestrepo — APPROVED** [review]
The changes look reasonable to me, but I wonder why the tests dis not run?

**Comment by mirestrepo** [comment]
oh... I remember now - I think they have to be manually triggered. This is because this runs on internal runner, but repo is public so we don't want changes from PRs to run automatically

**Review by mirestrepo — CHANGES_REQUESTED** [review]
You'll need to update the version of the docker and the path. See maybe the project module for related updates

**Comment by broestls** [comment]
I think I've resolved the requested issues, but I am spending a lot more time than I'd like trying to resolve issues with getting the `dorny/paths-filter` action to run. Probably will need some assistance there.

File: .github/workflows/kitchen-tests.yml [hunk 1]
```diff
@@ -5,53 +5,69 @@ on: [workflow_dispatch]
...
-    - name: Remove Terraform File
-      run: rm -rf examples/production-record/terraform.tfstate.d
-    - name: Post-check dirty dirs in self-hosted
-      run: |
-        ls -la 
-        ls -la examples/production-record/
+          TF_VAR_github_organization: BrownUniversity
+          TF_VAR_github_token: ${{ secrets.GH_token }}
+          INFOBLOX_USERNAME: ${{ secrets.INFOBLOX_JHUB_USER}}
+          INFOBLOX_PASSWORD: ${{ secrets.INFOBLOX_JHUB_USER}}
```
Comment (mirestrepo) [inline .github/workflows/kitchen-tests.yml:60]: `INFOBLOX_PASSWORD` is incorrectly set to `${{ secrets.INFOBLOX_JHUB_USER}}`

**Review by mirestrepo — CHANGES_REQUESTED** [review]
Can you also document in the README the environment variables that need to be set for the infoblox provider to work?

**Pull request merged**